data/presets/amenity/parcel_locker.json

```diff
@@ -24,7 +24,8 @@
         "mail",
         "packstation",
         "parcel",
-        "pickup"
+        "pickup",
+        "package locker"
```

Comment:
This is the array of `terms`, but you can add it to a separate array of `aliases` so that the user will see Package Locker as a result when they search for “package”.

Also, this repository’s contents more or less represent the American English localization, so if “parcel locker” is less common than “package locker” in this dialect, then “Package Locker” should be the preset name, relegating “Parcel Locker” to an alias.

Reply:
Ah, yeah that sounds good I can move it to there. 

I can't speak for the whole US, but for the parts that I have lived in (Pittsburgh, Houston, and Atlanta) 'Package' is a lot more common for folks to say than 'Parcel'. If you think it would be better to switch the name to Package Locker I can do that and add Parcel Locker as the alias instead.

Reply:
Fixed your first point, and left the second one open for discussion in case any one else wants to chime in on it.

Reply:
In general, I would agree about “package” being a more familiar term than “parcel”. I just wasn’t sure if that translated to lockers as well, but it sounds like it does. For what it’s worth, the USPS uses “[parcel locker](https://faq.usps.com/s/article/What-is-a-Cluster-Box)” to refer to a part of a cluster mailbox that doesn’t meet the definition of an `amenity=parcel_locker`. But they also tend to use the term “parcel” where most people would say “package”.